core/src/androidTest/java/novoda/lib/sqliteprovider/util/DBUtilsTest.java

```diff
@@ -71,6 +72,14 @@ public void testGettingUniqueConstraints() throws Exception {
         MoreAsserts.assertContentsInAnyOrder(constrains, new Constraint(Arrays.asList("const")));
     }
 
+    public void testGettingUniqueConstraintForIntegerPrimaryKey() throws Exception {
+        android.database.DatabaseUtils.createDbFromSqlStatements(getContext(), DB_NAME, 1, CREATE_TABLE_WITH_INTEGER_PRIMARY_KEY);
+
+        SQLiteDatabase db = getContext().openOrCreateDatabase(DB_NAME, 0, null);
+        List<Constraint> constrains = DBUtils.getUniqueConstraints(db, "t");
+        MoreAsserts.assertContentsInAnyOrder(constrains, new Constraint(Arrays.asList("_id")));
```

Comment:
Heh. C+P'd and adapted one of the other tests, that used to use a method called "getUniqueConstrains". I've fixed them all :)